explainers/content-addressable-bundles.md

```diff
@@ -425,9 +450,13 @@ formal procedure, however, there are several possible approaches:
   space-consuming.
 
   For example, given that browser has the following dependency graph of bundles,
-  where the bodies of `bundle-C`, `bundle-D`, `bundle-E` are missing, the
-  browser can start to fetch them again, possibly in parallel, without waiting
-  fetching `bundle-C`.
+  where the bodies of `bundle-C`, `bundle-D`, `bundle-E` are omitted to save the
+  cache disk space, the browser can start to fetch them again, possibly in
+  parallel. Note that a browser can start to fetch `bundle-D` and `bundle-E`
+  immediately without waiting fetching `bundle-C` (and scanning its index
```

Comment:
My understanding is:
 - Web developers should write multiple webbundle link elements in the HTML to fetch the webbundles in parallel if the loading performance is important.
 - This nested webbundle format should be used when lazy-loading is important, and the loading performance is not important.
 
Is my understanding correct?

If so, I don't understand the motivation of browser developers of introducing a new cache mechanism to keep the dependency information. Could you please explain this?

Reply:
A good question!

It's fine for a browser to prefetch nested bundles even if a user doesn't request it explicitly.

- If a user requests prefetch via `<link>` elements explicitly,  browser *should* prefetch it.
- If a user doesn't request it via `<link>` elements, a browser doesn't have to prefetch it, but *may* prefetch it, enabling the content to load instantly if and when the user requests it.